includes/admin/tools/class-edd-tools-recount-download-stats.php

```diff
@@ -77,45 +77,13 @@ public function get_data() {
 			)
 		), '3.0' );
 
-		global $wpdb;
-
-		/*
-		 * Build up our WHERE clauses.
-		 */
-		$conditions = array();
-
-		if ( ! empty( $accepted_statuses ) && is_array( $accepted_statuses ) ) {
-			$placeholder_string = implode( ', ', array_fill( 0, count( $accepted_statuses ), '%s' ) );
-			$conditions[]       = $wpdb->prepare(
-				"oi.status IN({$placeholder_string})",
-				$accepted_statuses
-			);
-		}
-
 		if ( ! empty( $this->download_id ) && is_numeric( $this->download_id ) ) {
-			$conditions[] = $wpdb->prepare(
-				"oi.product_id = %d",
-				$this->download_id
-			);
-		}
-		$conditions = ! empty( $conditions ) ? ' AND ' . implode( ' AND ', $conditions ) : '';
+			edd_recalculate_download_sales_earnings( $this->download_id );
```

Comment:
I removed a lot here as this tool can really only run on one download. The original code added a condition for the download ID if it wasn't empty but then updated the download meta potentially on a null ID.